File: src/main/java/online/cll/registry/itemgroup/ModItemGroups.java
package online.cll.registry.itemgroup;

import net.minecraft.item.Item;
import net.minecraft.item.ItemGroup;
import net.minecraft.item.ItemStack;
import net.minecraft.text.Text;
import net.minecraft.util.Rarity;
import online.cll.registry.item.ModItem;

public class ModItemGroups {
    public static final ItemGroup touhou_project =
            ModItemGroup.getItemGroup("touhou_project",
                    new ItemStack(ModItem.registry("group-icon",new Item(new Item.Settings().rarity(Rarity.EPIC)))),
                    Text.translatable("item-group.touhou-project-mod.touhou_project"));


    public static void registryItemGroup(){

    }
}

File: src/main/java/online/cll/registry/itemgroup/ModItemGroups.kt
package online.cll.registry.itemgroup

import net.minecraft.item.Item
import net.minecraft.item.ItemGroup
import net.minecraft.item.ItemStack
import net.minecraft.text.Text
import net.minecraft.util.Rarity// 稀有度
import online.cll.registry.item.ModItem

object ModItemGroups {
    val TOUHOU_PROJECT: ItemGroup = ModItemGroup.registry("touhou_project",
            ItemStack(ModItem.registry("group-icon", Item(Item.Settings().rarity(Rarity.EPIC)))),
            Text.translatable("item-group.touhou-project-mod.touhou_project"))

    fun registryItemGroup() {}
}
